lib/src/base-styles/_nx-btn.scss

```diff
@@ -135,9 +135,22 @@
 }
 
 .nx-btn--icon-only {
-  @extend .nx-btn--tertiary;
-
+  background-color: transparent;
   border-color: transparent;
+  color: var(--nx-color-text-dark);
+
+  &:hover {
+    background-color: var(--nx-swatch-grey-90);
+  }
+
+  &:focus:not(:active) {
+    outline: 2px solid var(--nx-color-interactive-border-focus);
+    outline-offset: -2px;
+  }
+
+  &:active {
+    background-color: var(--nx-swatch-grey-95);
```

Comment:
Need to confirm with @JasonDSwearingen , but it looks like Lists also have the same active background color of `grey-95`. In that case we'll need to update the `--nx-color-interactive-background-active` variable to be `grey-95` instead of `blue-95`.

Reply:
> Need to confirm with @JasonDSwearingen , but it looks like Lists also have the same active background color of `grey-95`. In that case we'll need to update the `--nx-color-interactive-background-active` variable to be `grey-95` instead of `blue-95`.

Updated the variable here after you confirmed with Jason: [10665ef](https://github.com/sonatype/sonatype-react-shared-components/pull/990/commits/10665ef8ef0855f0098a5a91672d2c362ad86bac)